R/UK.R

```diff
@@ -14,35 +14,35 @@
 #' @concept dataset
 #' @examples
 #' \dontrun{
-#' # Setup a data cache
+#' # setup a data cache
 #' start_using_memoise()
 #'
-#' # Download, clean and process level 1 UK data with hospital admissions
+#' # download, clean and process level 1 UK data with hospital admissions
 #' region <- UK$new(level = "1", nhsregions = TRUE)
 #' region$return()
 #'
-#' # Initialise level 2 data
+#' # initialise level 2 data
 #' utla <- UK$new(level = "2")
 #'
```

Comment:
Not relevant to the non-change here, but is there any way to make `DataClass$new` not crash if you feed it `2` as a level instead of `"2"`? I was working on expanding the wiki documentation of adding data classes and realized that I was managing to crash out with what seemed like the simplest invocation of `new`, all because I hadn't put quotes around a number. This felt like a very R problem.

Reply:
yes that is a good idea. I've added it to this PR. Other ease of use changes like this would be good to flag.